src/chunk.c

```diff
@@ -3027,30 +3016,12 @@ chunk_tuple_delete(TupleInfo *ti, DropBehavior behavior, bool preserve_chunk_cat
 
 	ts_catalog_database_info_become_owner(ts_catalog_database_info_get(), &sec_ctx);
 
-	if (!preserve_chunk_catalog_row)
-	{
-		ts_catalog_delete_tid(ti->scanrel, ts_scanner_get_tuple_tid(ti));
+	ts_catalog_delete_tid(ti->scanrel, ts_scanner_get_tuple_tid(ti));
 
-		if (form.dropped)
-			res = CHUNK_DELETED_DROPPED;
-		else
-			res = CHUNK_DELETED;
-	}
+	if (form.dropped)
```

Comment:
this column should be removed from the schema/catalog/etc as well - to not leave pollution behind...